src/contractIntegration/IPFSManagementInteraction/IPFSManagementInteraction.ts

```diff
@@ -1,3 +1,5 @@
+/*eslint no-unused-vars: "error"*/
+//eslint-disable-next-line
```

Comment:
@HectorGerardoZV Specify what type of error was ignored

Reply:
I added //eslint-disable-next-line, since it is used @injectable() and does not recognize it, so it takes it as not used {injectable} variable.